merkletree/str.go

```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/coniks-sys/coniks-go/utils"
 )
 
-// SignedTreeRoot represents a signed tree root, which is generated
+// SignedTreeRoot represents a signed tree root (STR), which is generated
 // at the beginning of every epoch.
 // Signed tree roots contain the current root node,
 // the current and previous epochs, the hash of the
 // previous STR, and its signature.
-// STR should be final
+// STR should be read-only.
```

Comment:
This seems to imply that the developer should ensure that the STR is read-only. It might be helpful to be more specific as to why/how the developer can ensure this.